src/test/java/seedu/academydirectory/logic/commands/RetrieveCommandTest.java

```diff
@@ -33,12 +32,10 @@
     @Test
     public void equals() {
         InformationWantedFunction firstFunction = new InformationWantedFunction(PREFIX_EMAIL);
-        InformationWantedFunction secondFunction = new InformationWantedFunction(PREFIX_ADDRESS);
         InformationWantedFunction thirdFunction = new InformationWantedFunction(PREFIX_TELEGRAM);
```

Comment:
I think you can rename this to secondFunction instead

Reply:
Yep changed in the next commit